tests/test_app.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from algorithm_services.config import config
+from algorithm_services.app import create_app
+
+from flask import Flask
+
+
+class AppTestCase(unittest.TestCase):
+
+    def setUp(self):
+        _app = create_app(__name__, config)
+        self.app = _app.test_client()
+        self.app.testing = True
+
+    def test_create_app(self):
+        self.assertTrue(isinstance(self.app.application, Flask))
```

Comment:
you should only test the app created by ´create_app´ method

Reply:
why are you testing test_app instance?

Reply:
As it is wrapped, app.application is the app itself, the wrapper is disposable for this tests, so i'm unwrapping the application.

Reply:
Should I test, anything else rather than if the app is a Flask instance?